### DISCUSSION TITLE
chatgpt continue generating button ?

### DISCUSSION THREAD

**Discussion by zdwork** [danny-avila/LibreChat#415] [Feature Requests & Suggestions]
<img width="684" alt="image" src="https://github.com/danny-avila/chatgpt-clone/assets/24469374/f01659be-d53a-4eee-8fd6-f23f79266712">

This function is very useful, hope to support

**Comment by fuegovic** [comment]
I added this to the feature request list

**Reply by zdwork** [reply]
There is no way to achieve it?

**Comment by danny-avila** [comment]
repeat of #380
